client/document/domains-landing.jsx

```diff
@@ -0,0 +1,177 @@
+/**
```

Comment:
The duplication between this and `index.jsx` worries me. Code tends to have a tendency to grow stale in these situations. In particular, I can see changes to the `evergreen` check being needed as we get more experience with odd customer setups in the wild, and there's a fair chance someone will forget to update one of the files.

I'm not super familiar with this part of the codebase, though, so I'm not entirely sure how best to handle reuse. @blowery: What do you think is the right approach here?

Reply:
I was actively holding off on trying to remove the duplication for a while. I share your concerns about these bits getting stale, but I'd like to get a handle on what the shared bits really are before we start consolidating.